python/lsst/ts/genericcamera/utils.py

```diff
@@ -22,10 +22,91 @@
 __all__ = [
     "DATE_FORMAT",
     "DATETIME_FORMAT",
+    "get_dayobs",
+    "make_image_names",
+    "parse_key_value_map",
 ]
 
+from astropy.time import Time, TimeDelta
+
+
 DATE_FORMAT = "%Y-%m-%d"
 """Format string for date values in the FITS header."""
 
 DATETIME_FORMAT = f"{DATE_FORMAT}T%H:%M:%S"
 """Format string for datetime values in the FITS header."""
+
+
+def get_dayobs(timestamp: float) -> str:
+    """Get the DAYOBS for the given timestamp.
+
+    Parameters
+    ----------
+    timestamp: `float`
+        The timestamp to derive the DAYOBS. Scale is assumed to be UTC.
+
+    Returns
+    -------
+    dayobs: `str`
+        The calculated DAYOBS.
+    """
+    time_obj = Time(timestamp, scale="utc", format="unix")
+    dayobs_time = time_obj - TimeDelta(12 * 3600, format="sec")
+    return dayobs_time.strftime("%Y%m%d")
+
+
+def make_image_names(gctag: str, dayobs: str, seqnums: list[int]) -> list[str]:
```

Comment:
how about `gc_tag`, `day_obs` and `seq_nums`?

Reply:
Done.